uniflow/op/model/model_config.py

```diff
@@ -43,6 +43,7 @@ class HuggingfaceModelConfig(ModelConfig):
     model_name: str = "mistralai/Mistral-7B-Instruct-v0.1"
     batch_size: int = 1
     model_server: str = "HuggingfaceModelServer"
+    neuron: bool = False
```

Comment:
In the HEAD of main now, we support `load_in_4bit` and `load_in_8bit` as below. If I recall correctly, you said taht neuron does no support quantization, so this can be a confusing Config for the user who might want to use neuron with quantization?
"""
@dataclass
class HuggingfaceModelConfig(ModelConfig):
    model_name: str = "mistralai/Mistral-7B-Instruct-v0.1"
    batch_size: int = 1
    model_server: str = "HuggingfaceModelServer"
    load_in_4bit: bool = False
    load_in_8bit: bool = True
"""

Reply:
I've added a logic that will tell the user.
```
if self._model_config.load_in_4bit or self._model_config.load_in_8bit:
    self._model_config.load_in_4bit = False
    self._model_config.load_in_8bit = False
    print(
        "Neuron model does not support quantized models. load_in_4bit and load_in_8bit are automatically set to False."
    )
```